Examples/Tests/energy_conserving_thermal_plasma/inputs_2d_electrostatic

```diff
@@ -0,0 +1,90 @@
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 500
+amr.n_cell =  8 8
+amr.max_level = 0
+geometry.dims = 2
+
+warpx.do_electrostatic = labframe
+algo.field_gathering = energy-conserving
+algo.particle_shape = 2
+warpx.use_filter = 0
+
+#################################
+############ CONSTANTS #############
+#################################
+
+my_constants.n0 = 1.e30          # plasma densirty, m^-3
+my_constants.Ti = 100.           # ion temperature, eV
+my_constants.Te = 100.           # electron temperature, eV
+my_constants.wpe = q_e*sqrt(n0/(m_e*epsilon0))  # electron plasma frequency, radians/s
+my_constants.de0 = clight/wpe    # skin depth, m
+my_constants.dt = ( 0.2 )/wpe        # time step size, s
+warpx.const_dt = dt
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+
+geometry.dims = 2
+geometry.prob_lo = 0.0 0.0
+geometry.prob_hi = 10.*de0 10.*de0
+boundary.field_lo = periodic periodic
+boundary.field_hi = periodic periodic
+boundary.particle_lo = periodic periodic
+boundary.particle_hi = periodic periodic
+
+#################################
+####### Boundary condition ######
+#################################
+boundary.field_lo = periodic periodic
+boundary.field_hi = periodic periodic
+
+#################################
+############ NUMERICS ###########
+#################################
+warpx.serialize_initial_conditions = 1
+warpx.verbose = 1
```

Comment:
I think that these parameters could be considered "general parameters" rather then "numerics" (they are not technical parameters of the solvers used in WarpX)